### PR TITLE
general-concepts/use-flags: remove redundant 'die' from the example

### PR DESCRIPTION
**Pull request by steils**
Signed-off-by: Stefan Strogin <steils@gentoo.org>

### CODE REVIEW

File: general-concepts/use-flags/text.xml [hunk 1]
```diff
@@ -276,14 +276,14 @@ src_compile() {
 	elif use ssl &amp;&amp; ! use gnutls ; then
 		myconf="${myconf} --enable-ssl --with-ssl=openssl"
 	else
-	myconf="${myconf} --disable-ssl"
+		myconf="${myconf} --disable-ssl"
```
Comment (ulm) [inline general-concepts/use-flags/text.xml]: All these assignments could profit from using +=, e.g. `myconf+=" --disable-ssl"`

Comment (steils) [reply general-concepts/use-flags/text.xml]: Why `+=`? There is no previous assignment in this example.
I think the example should be rewritten if we want to show profit from using `+=`. But that is for another PR.

Comment (ulm) [reply general-concepts/use-flags/text.xml]: Sure, we can be strict and split issues into different PRs. But then that indentation fix doesn't belong here either. :stuck_out_tongue:

Comment (steils) [reply general-concepts/use-flags/text.xml]: Sure:D

**Comment by ulm** [comment]
Pushed.

**Pull request closed without merging**